modules/mod_oauth/models/m_oauth_perms.erl

```diff
@@ -83,6 +83,8 @@ insert_all([], _Id, _Context) ->
     ok;
 insert_all([[] | Rest], Id, Context) ->
     insert_all(Rest, Id, Context);
+insert_all([<<>> | Rest], Id, Context) ->
+    insert_all(Rest, Id, Context);
```

Comment:
Ah, now that I see the context, I get it. This is due to the list --> binary change in master. Good catch, thanks!